README.md

```diff
@@ -133,14 +133,143 @@ pipe = LTXPipeline.from_pretrained(
     "Lightricks/LTX-Video", torch_dtype=torch.bfloat16
 ).to("cuda")
 + pipe.load_lora_weights("my-awesome-name/my-awesome-lora", adapter_name="ltxv-lora")
-+ pipe.set_adapters(["ltxv-lora"], [1.0])
++ pipe.set_adapters(["ltxv-lora"], [0.75])
```

Comment:
Golden number?

Reply:
Since I haven't found the optimal training settings yet for the LoRA, using the full power of it at 1.0 leads to slightly worse quality outputs. This seems to strike a nice balance but ideally should be explored by the person who trained